docs/_sources/auto_examples/plot_fmri_data_example.rst.txt

```diff
@@ -615,9 +614,9 @@ spurious discoveries.
 
 .. rst-class:: sphx-glr-timing
 
-   **Total running time of the script:** (1 minutes 21.474 seconds)
+   **Total running time of the script:** (1 minutes 30.075 seconds)
```

Comment:
Same here

Reply:
I think it's part of the variability of the test. It's a difference of 5 seconds over 1 min; for me it's negligible.
From my mistake, it's just a rerun of the documentation; thus, there is not really an indication of the difference in time.

Reply:
OK.